.changes/footer-with-experiments.md

```diff
@@ -0,0 +1,11 @@
+EXPERIMENTS:
+
+Experiments are only enabled in alpha releases of Terraform CLI. The following features are not yet available in stable releases.
+
+- `terraform test` accepts a new option `-junit-xml=FILENAME`. If specified, and if the test configuration is valid enough to begin executing, then Terraform writes a JUnit XML test result report to the given filename, describing similar information as included in the normal test output. ([#34291](https://github.com/hashicorp/terraform/issues/34291))
+- The new command `terraform rpcapi` exposes some Terraform Core functionality through an RPC interface compatible with [`go-plugin`](https://github.com/hashicorp/go-plugin). The exact RPC API exposed here is currently subject to change at any time, because it's here primarily as a vehicle to support the [Terraform Stacks](https://www.hashicorp.com/blog/terraform-stacks-explained) private preview and so will be broken if necessary to respond to feedback from private preview participants, or possibly for other reasons. Do not use this mechanism yet outside of Terraform Stacks private preview.
+- The experimental "deferred actions" feature, enabled by passing the `-allow-deferral` option to `terraform plan`, permits `count` and `for_each` arguments in `module`, `resource`, and `data` blocks to have unknown values and allows providers to react more flexibly to unknown values. This experiment is under active development, and so it's not yet useful to participate in this experiment
+
+## Previous Releases
+
+For information on prior major and minor releases, refer to their changelogs:
```

Comment:
Isn't this supposed to be part of the `footer.md`? That was a point of confusion during our last attempt with Sarah.

Reply:
If you feel strongly about it I could make it so, but I wanted to go with the very simple option of just having two kinds of footers, one with experiments and one without. The reason this is not part of the previous-releases is that in the `nextminor` command of the script I auto-append the release log line and this is way easier if I just need to append to the top of the line versus using `sed` to operate somewhere in the middle of the document (especially with sed being different between macos and linux)

Reply:
That's fair - I don't feel particularly strongly about it. To be honest the file rename already makes it significantly better. 😅